backend/app/Actions/Teams/RemoveTeamMemberAction.php

```diff
@@ -23,13 +21,11 @@ public function __construct(
         $this->websiteRepository = $websiteRepository;
     }
 
-    public function execute(RemoveTeamMemberRequest $request): void
+    public function execute($userId): void
     {
-        $website = $this->websiteRepository->getById($request->websiteId());
+        $teamMember = $this->userRepository->getById($userId);
 
-        $teamMember = $this->userRepository->getById($request->memberId());
-
-        $this->websiteRepository->removeMemberFromWebsiteTeam($teamMember, $website->id);
+        $website = $this->websiteRepository->removeMemberFromWebsiteTeam($teamMember);
```

Comment:
what is going on? firstly you added this code, now you are deleting this, what is happening?